README.md

```diff
@@ -662,6 +662,10 @@ datasets:
       # For `completion` datsets only, uses the provided field instead of `text` column
       field:
 
+# If true, the datasets will not be shuffled and will keep their original order in `datasets`.
+# The same applies to the `test_datasets` option and the `pretraining_dataset` option. Default is false.
+not_shuffle_merged_datasets: false
```

Comment:
I think inverted configuration variables are harder to understand. It's probably easier to use `shuffle_merged_datasets: false` where the default is true.

Reply:
Done